.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "SpongeAPI"]
 	path = SpongeAPI
-	url = https://github.com/SpongePowered/SpongeAPI.git
+	url = https://github.com/ST-DDT/SpongeAPI.git
```

Comment:
ITs required to prove that the PR builds.
Travis-CI will fail to build it otherwise.
The PRs are squashed and merged locally, so it does not matter at all (IMO).